### PR TITLE
Navigation merge test

### PR DESCRIPTION
**Pull request by jasonexus**
This contains the posts that Della made and fixes from the navigation-frontend branch from the last PR I made. Let me know if there's any issues.

### CODE REVIEW

File: src/components/Post-Components/MakePost.tsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
+import { h } from 'preact'
```
Comment (dalidia) [inline src/components/Post-Components/MakePost.tsx]: this is not used anywhere, can you remove it?

**Review by dalidia — APPROVED** [review]

File: src/pages/Login.tsx [hunk 2]
```diff
@@ -24,17 +24,32 @@ export default class Login extends Component<Props, State> {
...
+    //     <h1>Welcome to TikTakToe</h1>
+    //     <p>Connect with your friends in a private manner.</p>
+    //     <p>Currently we only support login with Github</p>
+    //     <div>
+    //       <h2>Log in to your account</h2>
+    //       <Button onClick={signUpWithGithub} variant="contained">Sign up with Github</Button>
+    //       <Button onClick={signInWithGithub} variant="contained">Log in with Github</Button>
+    //     </div>
+    //   </div>
+    // );
```
Comment (dellahumanita) [inline src/pages/Login.tsx]: If you don't need this section anymore, can you please remove it?

**Review by dellahumanita — CHANGES_REQUESTED** [review]

File: src/components/Post-Components/MakePost.tsx (hunk 1 above)
Comment (jasonexus) [reply src/components/Post-Components/MakePost.tsx]: File should be removed now.

File: src/pages/Login.tsx (hunk 2 above)
Comment (jasonexus) [reply src/pages/Login.tsx]: Should be removed now.

**Review by dellahumanita — APPROVED** [review]

**Pull request merged**